CHANGELOG.md

```diff
@@ -27,13 +22,8 @@
 
   ([PR #942](https://github.com/alphagov/govuk-frontend/pull/942))
 
-🔧 Fixes:
```

Comment:
fyi I think people are leaving these in for others to easily copy and paste in the future, before we cut a release

not a big deal to remove them though

Reply:
is it useful to have one in every section?

Reply:
maybe not, feel free to propose a change to https://github.com/alphagov/govuk-frontend/blob/master/docs/contributing/CHANGELOG_TEMPLATE.md